### DISCUSSION TITLE
Desktop mode and Active Signal mode (Refresh Rate)

### DISCUSSION THREAD

**Discussion by nbeh1** [seerge/g-helper#1464] [General]
I have encountered an issue with the refresh rate settings in G-helper. When on battery G-helper only changes desktop mode to 60hz but not active signal mode. This causes mouse stuttering issues. When manually changing in windows settings, mouse doesn't stutter.

When using G-helper, auto mode or manual changing to 60hz:
![Screenshot 2023-10-13 213005](https://github.com/seerge/g-helper/assets/139955754/91e5a6b2-547c-4370-b61c-c51466daeb84)

When manually changing it from windows display settings:
![Screenshot 2023-10-13 213020](https://github.com/seerge/g-helper/assets/139955754/403bb727-c77d-48ea-9154-d97872dadcea)

Does anyone have a similar issue? or is there a fixed? 
I have the 2022 G15 GA503RM, bios 315

**Comment by seerge** [comment]
@nbeh1 hello, g-helper just uses windows api to set refresh rate of the screen.

If it doesn't work by some reason, I would advice to do clean GPU drivers reinstall (using DDU first)

**Reply by nbeh1** [reply]
I will try it!

**Reply by nbeh1** [reply]
I did a drivers reinstall with DDU cleanup, same issue. I used asus oem drivers.

**Comment by seerge** [comment]
@nbeh1 may be it's some strange monitor drivers ? anyway i can't really change anything here, as refresh rate setting is super straightforward  - you just set value (60Hz or anything else), and that's it :)

**Reply by nbeh1** [reply]
Yea I know, thanks for everything really! 
I'll just have to manually change it for now ：）
